app/controllers/laa_references_controller.rb

```diff
@@ -1,14 +1,36 @@
 # frozen_string_literal: true
 
 class LaaReferencesController < ApplicationController
+  include DefendantHelpers
+
+  before_action :set_defendant_identifier_if_required,
+                :load_and_authorize_search,
+                :set_link_attempt,
+                :set_defendant_if_required
+
+  add_breadcrumb :search_filter_breadcrumb_name, :new_search_filter_path
+  add_breadcrumb :search_breadcrumb_name, :search_breadcrumb_path
+
+  def new
+    add_defendant_case_breadcrumb
+    add_breadcrumb defendant.name,
+                   defendant_path(defendant.arrest_summons_number || defendant.national_insurance_number)
+  end
+
   def create
     authorize! :create, :link_maat_reference, message: I18n.t('unauthorized.default')
-    if link_laa_reference
-      flash[:notice] = I18n.t('laa_reference.link.success')
+
+    if @link_attempt.valid?
+      if link_laa_reference
+        flash[:notice] = I18n.t('laa_reference.link.success')
+        redirect_to edit_defendant_path(@defendant_identifier)
+      else
+        flash[:alert] = I18n.t('laa_reference.link.failure', error_messages: error_messages)
+        redirect_to new_laa_reference_path(id: @defendant_identifier)
+      end
     else
-      flash[:alert] = I18n.t('laa_reference.link.failure', error_messages: error_messages)
+      render :new
```

Comment:
here you use symbol, other controller uses string. i'd choose one and apply to both, for consistency

Reply:
👍 I have now used string in both controllers